CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.12.15+8
```

Comment:
Unfortunate, but intended. Ideally the pubspec would be `0.12.15+8-dev`, but that's not allowed by semver, so we use the next-lowest dev version instead. If we release before adding any new features, it'll match the changelog.